### ISSUE TITLE
Return code documentation describes symbols instead of enum

### ISSUE THREAD

**Issue by thanasibakis** [SciML/DiffEqDocs.jl#734]
Hi,

The documentation for solution return codes at https://docs.sciml.ai/DiffEqDocs/stable/basics/solution/#retcodes describes symbols as return codes, which I believe is out of date and does not match the enum implementation described at https://docs.sciml.ai/SciMLBase/stable/interfaces/Solutions/#retcodes?

**Comment by ChrisRackauckas** [comment]
Yes thank you for pointing that out. That is out of date and it should be updated to point to the return codes there.
